src/AspNet.Security.OAuth.HubSpot/AspNet.Security.OAuth.HubSpot.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Metadata for the package is missing. For example:

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/5f311082a46ed519d0d6f7428e7018870657ebe9/src/AspNet.Security.OAuth.Amazon/AspNet.Security.OAuth.Amazon.csproj#L7-L11